pom.xml

```diff
@@ -34,7 +34,7 @@
     <inceptionYear>2019</inceptionYear>
 
     <properties>
-        <easy.schema.version>3.3.1</easy.schema.version>
+        <easy.schema.version>3.3.2-SNAPSHOT</easy.schema.version>
```

Comment:
To be fixed once easy-schema is merged and available via maven. That will fix the build.